docs/SETUP.md

```diff
@@ -139,9 +150,7 @@ helm repo update
 helm install nvidia/gpu-operator \
   --wait --generate-name \
   --create-namespace -n gpu-operator \
-  --set driver.enabled=false \
-  --set mig.strategy=none \
-  --version 23.3.1
+  --set driver.enabled=false
```

Comment:
Are we sure it is OK to remove:
```
--set mig.strategy=none \
--version 23.3.1
```
Missing these two options caused many problems for us in the past.

Reply:
It has worked without those options for me in both MIG and non-MIG environments on different machines.
Ir-respective for environments which have GPUs with MIG enabled, the key seems to be to create GPU MIG instances prior to installing the operator other-wise the CUDA validator fails.